pylossless/pipeline.py

```diff
@@ -565,11 +535,19 @@ def chan_neighbour_r(epochs, nneigbr, method):
         this_ch_xr = xr.DataArray([this_ch * np.ones_like(nearest_chs)],
                                   dims=['ref_chan', 'epoch',
                                         'channel', 'time'],
-                                  coords={'ref_chan': [name]})
+                                  coords={'ref_chan': [name],
```

Comment:
As I told you before, there were some dimensions that didn't have coordinates. I added the coordinates because I needed to access them during debugging.